### PR TITLE
chore(api-docs): Add documentation for GroupEventsEndpoint

### PR DESCRIPTION
**Pull request by malwilley**
Ref https://github.com/getsentry/getsentry/issues/13865

Before:

CleanShot 2024-10-03 at 13 47 14

After:

CleanShot 2024-10-03 at 13 49 39

### CODE REVIEW

File: src/sentry/apidocs/hooks.py [hunk 1]
```diff
@@ -297,27 +297,26 @@ def _fix_issue_paths(result: Any) -> Any:
...
     for path, endpoint in items:
         if "{var}/{issue_id}" in path:
-            if path.startswith("/api/0/organizations/{organization_id_or_slug}/"):
-                deleted_paths.append(path)
-            else:
-                modified_paths.append(path)
-
-    for path in deleted_paths:
-        del result["paths"][path]
+            modified_paths.append(path)
```
Comment (malwilley) [inline src/sentry/apidocs/hooks.py:304]: Added this hook in https://github.com/getsentry/sentry/pull/78483 to make it possible to document the `/issues|groups/{issue_id}` endpoints. Didn't test it with the real docs though so it needed a small change to work correctly

File: src/sentry/issues/endpoints/group_events.py [hunk 2]
```diff
@@ -38,29 +50,60 @@ class GroupEventsError(Exception):
...
+            IssueParams.ISSUE_ID,
+            GlobalParams.START,
+            GlobalParams.END,
+            GlobalParams.STATS_PERIOD,
+            GlobalParams.ENVIRONMENT,
+            OpenApiParameter(
+                name="full",
+                type=OpenApiTypes.BOOL,
+                location=OpenApiParameter.QUERY,
+                description="If this is set to true, the event payload will include the full event body, including the stacktrace. Set to true to enable.",
```
Comment (isabellaenriquez) [inline src/sentry/issues/endpoints/group_events.py:75]: nit: probably don't need the second sentence? maybe reword to "Specify `true` to include the full event body, including the stacktrace, in the event payload." to match existing documentation of bool params

File: src/sentry/issues/endpoints/group_events.py [hunk 3]
```diff
@@ -38,29 +50,60 @@ class GroupEventsError(Exception):
...
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=EventExamples.GROUP_EVENTS_SIMPLE,
+    )
     def get(self, request: Request, group: Group) -> Response:
         """
-        List an Issue's Events
-        ``````````````````````
-
         This endpoint lists an issue's events.
```
Comment (isabellaenriquez) [inline src/sentry/issues/endpoints/group_events.py:106]: nit: to mimic more recent docs, maybe use "Return a list of error events bound to an issue" from the OpenApiExample

**Review by isabellaenriquez — APPROVED** [review]
just some wording nits, lgtm overall!
